packages/wallet/src/services/types.ts

```diff
@@ -71,8 +73,23 @@ export interface Delegatee {
   nextNextEpoch: Cardano.StakePool | null;
 }
 
+export enum DelegationKeyStatus {
+  Registering = 'REGISTERING',
+  Registered = 'REGISTERED',
+  Unregistering = 'UNREGISTERING',
+  Unregistered = 'UNREGISTERED'
+}
+
+export interface RewardAccount {
+  address: Cardano.Address;
+  keyStatus: DelegationKeyStatus;
+  // Maybe add rewardsHistory$ and delegatee$ for each reward account too
```

Comment:
:+1:

Reply:
I think we should leave this out of scope of this PR and do it when we need multi-address support